src/main/kotlin/io/github/rybalkinsd/kohttp/pool/CoroutineExecutorService.kt

```diff
@@ -0,0 +1,43 @@
+package io.github.rybalkinsd.kohttp.pool
+
+import kotlinx.coroutines.*
+import org.jetbrains.spek.meta.Experimental
+import java.util.concurrent.Executor
+import java.util.concurrent.ExecutorService
+import java.util.concurrent.ForkJoinPool
+import kotlin.coroutines.CoroutineContext
+
+/**
+ * @Author: evgeny.vorobyev
+ * @Since: 0.10.0
+ */
+@Experimental
+class CoroutineExecutorService(
```

Comment:
it is better to move experimental changes to the separate branch

Reply:
will make it later